lib/rmt/http_request.rb

```diff
@@ -5,14 +5,14 @@ class RMT::HttpRequest < Typhoeus::Request
 
   def set_defaults
     Typhoeus::Config.user_agent = "RMT/#{RMT::VERSION}"
-    Typhoeus::Config.verbose = Settings.http_client.verbose || nil
+    Typhoeus::Config.verbose = Settings.try(:http_client).try(:verbose) || nil
```

Comment:
The ` || nil` at the end should not be required anymore, right?

Reply:
Right!